webpack/assets/javascripts/react_app/components/StatisticsChartsList/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ConnectedChartBox from '../ChartBox/ConnectedChartBox';
+import './StatisticsChartsListStyles.scss';
+import { translate as __ } from '../../common/I18n';
+
+const StatisticsChartsList = ({ data }) => {
+  const chartBoxes = Object.values(data).map(chart => (
+    <ConnectedChartBox
+      key={chart.id}
+      type="donut"
+      chart={chart}
+      noDataMsg={__('No data available')}
+      tip={__('Expand the chart')}
+    />
+  ));
+
+  const isEmpty = !(data && chartBoxes.length);
+  return (
+    <div className="statistics-charts-list-root cards-pf">
+      {isEmpty ? <p>{__('No Charts Loaded')}</p> : chartBoxes}
+    </div>
+  );
+};
```

Comment:
awesome work on this file :+1: 
can you return early if there's no data?

e.g: 
```js
if ( !data ) {
  return <p>{__('No Charts Loaded')}</p>
} 

return Object.values(data).map(chart => (
￼    <ConnectedChartBox
￼      key={chart.id}
￼      type="donut"
￼      chart={chart}
￼      noDataMsg={__('No data available')}
￼      tip={__('Expand the chart')}
￼    />
￼  ));
```
feels more readable,
another thought - should this code move to `routes/Statistics/StatisticsPage/Statistics/Statistics.js` ?

Reply:
I agree, it makes more sense that way